tests/unit/amo/components/TestCollection.js

```diff
@@ -66,6 +66,23 @@ describe(__filename, () => {
     expect(wrapper.find('.Collection-wrapper')).toHaveLength(1);
   });
 
+  it('allows HTML entities in the Collection description', () => {
+    const { store } = dispatchClientMetadata();
+
+    // We need a collection for this test case.
```

Comment:
I know you got this from another test but I don't think it's really helpful. It should be pretty obvious (from the code) that the test needs a collection.

Reply:
Ah, indeed. Just a bad copypasta. Thanks 👍